charts/sonarqube-dce/templates/secret.yaml

```diff
@@ -14,32 +28,54 @@ data:
   {{ template "jdbc.secretPasswordKey" . }}: {{ template "jdbc.internalSecretPasswd" . }}
 {{- end }}
 
+{{- if eq (include "jwtSecret.useInternalSecret" .) "true" }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "sonarqube.fullname" . }}-monitoring-passcode
+  name: {{ template "sonarqube.fullname" . }}-jwt
   labels:
     app: {{ template "sonarqube.name" . }}
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  SONAR_WEB_SYSTEMPASSCODE: {{ .Values.monitoringPasscode | b64enc | quote }}
+  SONAR_AUTH_JWTBASE64HS256SECRET: {{ .Values.ApplicationNodes.jwtSecret | b64enc | quote }}
+{{- end }}
 
-{{- if eq (include "jwtSecret.useInternalSecret" .) "true" }}
+{{- if .Values.searchNodes.searchAuthentication.enabled }}
+
+{{- if eq (include "search.useInternalKeystoreSecret" .) "true" }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "sonarqube.fullname" . }}-jwt
+  name: sonarqube-keystore-pass
   labels:
     app: {{ template "sonarqube.name" . }}
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  SONAR_AUTH_JWTBASE64HS256SECRET: {{ .Values.ApplicationNodes.jwtSecret | b64enc | quote }}
+  SONAR_CLUSTER_ES_SSL_KEYSTOREPASSWORD: {{ .Values.searchNodes.searchAuthentication.keyStorePassword | b64enc | quote }}
+  SONAR_CLUSTER_ES_SSL_TRUSTSTOREPASSWORD: {{ .Values.searchNodes.searchAuthentication.keyStorePassword | b64enc | quote }}
```

Comment:
As I said in previous comment, I think we should have different properties for keystore and truststore password